File: content-solutions/sorting/meeting-rooms.js
function solution(meetings) {
  const events = [];
  for (const [s, e] of meetings) events.push([s, 1], [e, -1]);
  events.sort((a, b) => a[0] - b[0] || a[1] - b[1]);
  let now = 0;
  let best = 0;
  for (const [, d] of events) {
    now += d;
    best = Math.max(best, now);
  }
  return best;
}

File: content-solutions/sorting/meeting-rooms.py
def solution(meetings):
    events = []
    for s, e in meetings:
        events.append((s, 1))
        events.append((e, -1))
    events.sort()
    now = best = 0
    for t, d in events:
        now += d
        best = max(best, now)
    return best
